packages/request-client.js/src/http-request-network.ts

```diff
@@ -24,23 +25,34 @@ export default class HttpRequestNetwork extends RequestNetwork {
     {
       decryptionProvider,
       nodeConnectionConfig,
+      persistFromLocal,
       signatureProvider,
       useMockStorage,
+      web3,
+      urlProvider,
     }: {
       decryptionProvider?: DecryptionProviderTypes.IDecryptionProvider;
       nodeConnectionConfig?: AxiosRequestConfig;
       signatureProvider?: SignatureProviderTypes.ISignatureProvider;
       useMockStorage?: boolean;
+      persistFromLocal?: boolean;
+      web3?: any;
+      urlProvider?: string;
     } = {
       nodeConnectionConfig: {},
+      persistFromLocal: false,
```

Comment:
not conviced about this naming, but can't come up with a better one yet... anyone has a better idea?

Reply:
@rittme ? @ltacker ? Any suggestion ?

Reply:
`manualEthereumPersist`
`onlyIPFSNode`

Reply:
`directEthereumBroadcast` `ethereumBroadcast` `localEthereumBroadcast` ?

Reply:
I go for `useLocalEthereumBroadcast`. Feels right for you guys?

Reply:
`useLocalEthereumBroadcast` or just `localEthereumBroadcast` is also fine for me